packages/delegation-process/src/app.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  authenticationMiddleware,
+  contextMiddleware,
+  loggerMiddleware,
+  zodiosCtx,
+  buildJwksClients,
+} from "pagopa-interop-commons";
+
+import healthRouter from "./routers/HealthRouter.js";
+import delegationProducerRouter from "./routers/DelegationProducerRouter.js";
+import delegationRouter from "./routers/DelegationRouter.js";
+import { config } from "./config/config.js";
+
+const serviceName = "delgation-process";
+
+const app = zodiosCtx.app();
+
+const jwksClients = buildJwksClients(config);
+
+// Disable the "X-Powered-By: Express" HTTP header for security reasons.
+// See https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#recommendation_16
+app.disable("x-powered-by");
+app.use(contextMiddleware(serviceName));
+app.use(healthRouter);
```

Comment:
These two routers must be inverted, health must be before the context middleware, otherwise, we require the CorrelationId to be present in the health call (we don't want to, this already resulted in a bug before) - fixed in https://github.com/pagopa/interop-be-monorepo/pull/1083